src/Polly.Contrib.WaitAndRetry/Backoff.DecorrelatedJitter.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+
+namespace Polly.Contrib.WaitAndRetry
+{
+    partial class Backoff // .DecorrelatedJitter
+    {
+        /// <summary>
+        /// Generates sleep durations in an jittered manner, making sure to mitigate any correlations.
+        /// For example: 117ms, 236ms, 141ms, 424ms, ...
+        /// For background, see https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/.
+        /// </summary>
+        /// <param name="minDelay">The minimum duration value to use for the wait before each retry.</param>
+        /// <param name="maxDelay">The maximum duration value to use for the wait before each retry.</param>
+        /// <param name="retryCount">The maximum number of retries to use, in addition to the original call.</param>
+        /// <param name="fastFirst">Whether the first retry will be immediate or not.</param>
+        /// <param name="seed">An optional <see cref="Random"/> seed to use.
+        /// If not specified, will use a shared instance with a random seed, per Microsoft recommendation for maximum randomness.</param>
+        public static IEnumerable<TimeSpan> DecorrelatedJitterBackoff(TimeSpan minDelay, TimeSpan maxDelay, int retryCount, bool fastFirst = false, int? seed = null)
```

Comment:
Where other examples had more than one optional parameter, `bool fastFirst = false` always appeared last.  Should we make it consistent?  (No opinion which way round it should be fastFirst last or earlier)